hcx-core/hcx-common/src/main/java/org/swasth/common/utils/Constants.java

```diff
@@ -53,6 +53,8 @@
 
     //Request props
     public static final String PROTOCOL_HEADERS_MANDATORY = "protocol.headers.mandatory";
+    public static final String ERROR_HEADERS_MANDATORY = "error.headers.mandatory";
+    public static final String ERROR_HEADERS_OPTIONAL = "error.headers.optional";
```

Comment:
Change the name. Maybe "plainrequest.headers.mandatory"

Reply:
Done